index.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
need to list exported stuff here. As I understand it should be actionsSteps, verificationSteps, hooks?, parameterTypes

Reply:
done